Pipfile

```diff
@@ -15,6 +15,7 @@ mido = "*"
 python-rtmidi = "*"
 pretty-midi = "*"
 note-seq = "*"
+huggingface-hub = "*"
```

Comment:
don't we also need need pytorch for the generation ?

Reply:
hmmm. actually not :D
but we will need it for other things for sure